src/sass/layout/_main.scss

```diff
@@ -0,0 +1,520 @@
+        content: counter(listCounter);
+        font-size: var(--larger-font-size);
+        border: var(--list-bullet-border);
+        color: var(--list-bullet-color);
+        border-radius: 50%;
+        width: $list-el-bullet-square-size;
+        height: $list-el-bullet-square-size;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        line-height: 1.0;
+        flex-shrink: 0;
+        margin-right: $list-el-bullet-margin-right;
+    }
+} 
+
+.grid {
+
+    $grid-info-side-margin: 6.875rem;
+    $title-padding-bottom: 1.5rem;
+    $paragraph-max-width: 35rem;
+    $paragraph-padding-bottom: 3.75rem;
+
+    height: fit-content;
+    width: inherit;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    background-color: var(--gray-background);
+
+    &__img {
+        width: 100%;
+    }
+
+    &__info-top{
+        margin-left: $grid-info-side-margin;
+        margin-right: auto;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    &__info-bottom{
+        margin-right: $grid-info-side-margin;
+        margin-left: auto;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    &__title {
+        @include abstracts.gradient-text;
+        font-size: var(--main-title-font-size);
+        line-height:normal;
+        padding-bottom: $title-padding-bottom;
+    }
+
+    &__paragraph {
+        line-height: 2.25rem;
+        max-width: $paragraph-max-width;
+        color: var(--standard-paragraph-color);
+        text-align: left;
+        padding-bottom: $paragraph-padding-bottom;
+    }
+
+    &__button {
+        @include abstracts.btn-blue;
+        width: fit-content;
+    }
+}
+
+.sign-up {
+
+    $ideal-signup-height: 489;
+    $email-cover-radius-offset: 3rem;
+    $input-padding-left: 1.25rem;
+    $title-margin-bottom: 2.625rem;
+
+    width: inherit;
+    height: #{scaleHeight($ideal-signup-height)};    
+    background-image: var(--background-gradient);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: nowrap;
+
+    &__img {
+        
+        opacity: 0.2;
+        z-index: 0;
+        width: 100%;
+        position: absolute;   
+    }
+
+    &__title {
+        position: relative;
+        font-size: var(--main-title-font-size);
+        color: var(--white-text);
+        z-index: 100;
+        margin-bottom: $title-margin-bottom;
+    }
+
+    &__form {
+        position: relative;
+        width: fit-content;
+    }
+
+    &__input {
+        height: 100%;
+        color: var(--white-text);
+        border: var(--transparent-button-border);
+        background-color: var(--signup-input-background-color);
+        padding-left: $input-padding-left;
+        padding-right: calc($email-cover-radius-offset + $input-padding-left / 2);
+        border-radius: var(--border-radius);
+        outline: none;
```

Comment:
Omitting outline is not the best idea because it will be harder to navigate through the page with accessibility mode.